src/App.js

```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 import { Route } from 'react-router-dom';
 import Navigation from './components/Navigation/Navigation';
 import SongLibrary from './components/SongLibrary/SongLibrary';
+import MySongLibrary from './components/MySongLibrary/MySongLibrary';
```

Comment:
This line is throwing an error -> I think the directory starts with a lowercase: `mySongLibrary`